keyboards/idobo/keymaps/xaceofspaidsx/keymap.c

```diff
@@ -0,0 +1,88 @@
+  { KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_END,  KC_UP,   KC_PGDN, KC_N,     KC_M,     KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT  },
+  { KC_LCTL, KC_LGUI, KC_LALT, MO(_FN), KC_SPC,  KC_SPC,  KC_LEFT, KC_DOWN, KC_RGHT, KC_LBRC,  KC_RBRC,  MO(_FN), KC_RALT, KC_RGUI, KC_RCTL  },
+ },
+
+/* FUNCTION Layer - removed the rgb hue saturation and vibrance for audio controls and added some functionality i use from other boards.
+ * .--------------------------------------------------------------------------------------------------------------------------------------.
+ * | F1     | F2     | F3     | F4     | F5     | F6     | NUM LK | P/     | P*     | F7     | F8     | F9     | F10    | F11    | F12    |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * | SELECT | CALC   | UP     |        |        | NEXT   | P7     | P8     | P9     | -      |        |        | PR SCR | SCR LK | PAUSE  |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * | PREV   | LEFT   | DOWN   | RIGHT  | DEL    | PLAY   | P4     | P5     | P6     | +      |        | RESET  |        |        |        |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * | VOL-   | MUTE   | VOL+   | APP    |        | STOP   | P1     | P2     | P3     | PENT   |        |        |        |        |        |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * |        |        | RGB TG | FN     | RGB RMD| RGB MD | P0     |        | P.     | PENT   | PENT   | FN     |        |        |        |
+ * '--------------------------------------------------------------------------------------------------------------------------------------'
+ */
+ 
+ [_FN] = { /* FUNCTION */
+  { KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_NLCK, KC_SLSH, KC_ASTR, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12   },
+  { KC_MSEL, KC_CALC, KC_UP  , _______, _______, KC_MNXT, KC_P7,   KC_P8,   KC_P9,   KC_MINS, _______, _______, KC_PSCR, KC_SLCK, KC_PAUS  },
+  { KC_MPRV, KC_LEFT, KC_DOWN, KC_RGHT, KC_DEL , KC_MPLY, KC_P4,   KC_P5,   KC_P6,   KC_PLUS, _______, RESET,   _______, _______, _______  },
+  { KC_VOLD, KC_MUTE, KC_VOLU, KC_APP,  _______, KC_MSTP, KC_P1,   KC_P2,   KC_P3,   KC_PENT, _______, _______, _______, _______, _______  },
+  { _______, _______, RGB_TOG, MO(_FN), RGB_RMOD,RGB_MOD, KC_P0,   _______, KC_PDOT, KC_PENT, KC_PENT, MO(_FN), _______, _______, _______  },
+ }
+};
+const uint16_t PROGMEM fn_actions[] = {
+
+};
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt)
```

Comment:
This function has been superseded by `process_record_user`.

Your keymap isn't using this functionality, so this code block should be deleted.